### DISCUSSION TITLE
Installation Help Please

### DISCUSSION THREAD

**Discussion by Cironde** [elementary/triage#329] [Q&A]
To get this out of the way: yes,  I'm a noob. I already feel like an idiot so please be gentle.  Trust me, I've looked everywhere I can think of and haven't noticed an answer.  I've seen people ask the same basic question where I've seen just one answer and it involved just waiting and a graphical interface would pop up and all would be well. 
So,  I select the Try option.  It immediately comes back with a prompt coming after "elementary login". I've tried everything I can think of and of course nothing,  to no avail.  
Well,  actually,  if I put elementary as my login and no password I get what's in the 3rd picture. 
Could someone please just tell me what selection I'm doing wrong? I've been trying out various distros for the last few days (15 at last count) and besides a couple where one just wouldn't work with my nvidia card (feren) and another that didn't have a driver for my wireless card (garuda) I have had no issues. Elementary was in my top 3 of wanting to try and this is very frustrating. 
If I somehow missed the correct answer to someone else's question,  I apologize and hope you'll leave me a link. 

Thanks. 
![20230919_134138](https://github.com/elementary/triage/assets/145503605/213cb1b1-7d8b-496f-818a-9e885988f199)
![20230919_134232](https://github.com/elementary/triage/assets/145503605/3a38002a-750a-4760-b5bb-25fba822809c)
![20230919_140657](https://github.com/elementary/triage/assets/145503605/0ea7a80e-a9cd-46c7-9bbf-2f580dbbf161)

**Comment by IanHead** [comment]
I have noticed that sometimes that login prompt comes up for a few seconds when I boot up my live USB, but then it does come up with the installer screen afterwards. If you're stuck there at that login prompt, something might've gone wrong. 

Maybe give it a try with the safe graphics option? Or, if that doesn't work, then after you hit that tty prompt and login with `elementary` as the user and no password, can you type `startx` at the prompt and hit enter, and see if that gets you anywhere?

**Comment by Cironde** [comment]
Hi,

Thanks for the input.  I actually got tired of waiting and installed Nobara
and am having fun with it for now.  Perhaps if I get bored with it I will
go back and try your work- around.

Thanks again,

Scott

**Comment by Cironde** [comment]
hi guys,

so, it turns out that there must have been a problem with the original download. due to issues i had to download a new copy  and it does exactly what you said it should do. boots into the os after about 2 seconds.

thanks again,
scott


________________________________
